src/controllers/expensesController.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const { StatusCodes, ReasonPhrases } = require('http-status-codes');
+const expensesService = require('../services/expensesService');
+const usersService = require('../services/usersService');
+
+function get(req, res) {
+  const { userId, categories, from, to } = req.query;
+
+  res.status(StatusCodes.OK).send(
+    expensesService.get({
+      userId,
+      categories,
+      from,
+      to,
+    }),
+  );
+}
+
+function post(req, res) {
+  const data = req.body;
+
+  if (!usersService.getById(data.userId)) {
+    res
+      .status(StatusCodes.BAD_REQUEST)
+      .send(
+        `${ReasonPhrases.BAD_REQUEST}: user with id ${data.userId} does not exist`,
+      );
+
+    return;
+  }
+
+  res.status(StatusCodes.CREATED).send(expensesService.create(data));
```

Comment:
Similar to the `get` function, the `post` function directly sends the result of `expensesService.create(data)`. Consider adding error handling to manage potential errors from the service.